From: Heiko Becker <heirecka@exherbo.org>
Date: Fri, 07 Aug 2015 17:02:55 +0000
Subject: Fix building the qalculate backend
X-Git-Url: http://quickgit.kde.org/?p=cantor.git&a=commitdiff&h=ceadbd453cfb4263755931635cb9b02d7aa29efa
---
Fix building the qalculate backend

Thanks to Christoph Feck who suggested the fix.

BUG: 350729
REVIEW: 124829
---


--- a/src/backends/qalculate/qalculateexpression.cpp
+++ b/src/backends/qalculate/qalculateexpression.cpp
@@ -773,12 +773,12 @@
 	if (plotParameters.filetype == PLOT_FILETYPE_EPS ||
 	    plotParameters.filetype == PLOT_FILETYPE_PS  ||
 	    (plotParameters.filetype == PLOT_FILETYPE_AUTO && p >= 4 &&
-	     plotParameters.filename.substr(p-4,4) == QLatin1String(".eps")) ||
+	     plotParameters.filename.substr(p-4,4) == ".eps") ||
 	    (plotParameters.filetype == PLOT_FILETYPE_AUTO && p >= 3 &&
-	     plotParameters.filename.substr(p-3,3) == QLatin1String(".ps")))
+	     plotParameters.filename.substr(p-3,3) == ".ps"))
         setResult(new Cantor::EpsResult(QUrl(QString::fromStdString(plotParameters.filename))));
 	else
-        setResult(new Cantor::ImageResult(QUrl(QString::fromStdString(plotParameters.filename)));
+        setResult(new Cantor::ImageResult(QUrl(QString::fromStdString(plotParameters.filename))));
 #else
     setResult(new Cantor::ImageResult(QUrl::fromLocalFile(QString::fromStdString(plotParameters.filename))));
 #endif

